src/js/pages/jobs/JobDetailPage.js

```diff
@@ -245,10 +242,12 @@ class JobDetailPage extends mixin(TabsMixin) {
 
     return (
       <Page>
-        <Page.Header
+        <JobsMenu
```

Comment:
Please consider keeping `Page.Header` looks like the way @orlandohohmeier suggested in the thread https://mesosphere.slack.com/archives/C03UXUVH7/p1529066379000323 could work. With that we won't need to modify the TemplateUtil